Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.2.2'
+gem 'rack-cors'
```

Comment:
- [ ] I am getting a Cors policy error when I visit the backend API from the frontend app. Kindly fix this.

  You need to visit the `config\initializers\cors.rb` and uncomment lines 8 to 16 and change "example.com" to "*" to allow requests from all sources

  ![image](https://github.com/CaptainBawa/Hello-Rails-Back-End/assets/81468675/c9df3559-50b0-4b23-b6a1-b6b4d5de07f4)

- [ ] Kindly refer to the changes on the frontend PR.